src/hdrgen.d

```diff
@@ -2459,7 +2454,15 @@ public:
 
     override void visit(DeclarationExp e)
     {
-        e.declaration.accept(this);
+        if (auto v = e.declaration.isVarDeclaration())
+        {
+            buf.writeByte('(');
+            visitVarDecl(v, false);
+            buf.writeByte(';');
+            buf.writeByte(')');
```

Comment:
I read your comment several times and do not understand it. Are you saying that the code currently generates syntactically incorrect code? or not? or you fixed it so it does?

By the way, thanks for splitting up the PRs into digestible chunks. It makes things a lot easier.